packages/contracts-bedrock/scripts/upgrades/v1.8.0-permissioned/scripts/common.sh

```diff
@@ -0,0 +1,232 @@
+#
+# Returns:
+#   The address associated with the specified name
+#
+# Exits with status 1 if:
+#   - The deployments JSON file is not found
+#   - The specified address name is not found in the JSON file
+load_local_address() {
+    local deployments_json_path="$1"
+    local address_name="$2"
+    local alt_name="${3:-}"
+
+    if [ ! -f "$deployments_json_path" ]; then
+        echo "Error: Deployments JSON file not found: $deployments_json_path" 1>&2
+        exit 1
+    fi
+
+    local address
+    address=$(jq -r ".$address_name" "$deployments_json_path")
+
+    if [ -z "$address" ] || [ "$address" == "null" ]; then
+        if [ -n "$alt_name" ]; then
+            address=$(jq -r ".$alt_name" "$deployments_json_path")
+        fi
+
+        if [ -z "$address" ] || [ "$address" == "null" ]; then
+            echo "Error: $address_name not found in deployments JSON" 1>&2
+            exit 1
+        fi
+    fi
+
+    echo "$address"
+}
+
+# fetch_standard_address
+#
+# Fetches the implementation address for a given contract from a TOML file.
+# The TOML file is downloaded from a URL specified in ADDRESSES_TOML_URL
+# environment variable. Results are cached to avoid repeated downloads.
+#
+# Arguments:
+#   $1 - Network name
+#   $2 - The release version
+#   $3 - The name of the contract to look up
+#
+# Returns:
+#   The implementation address of the specified contract
+#
+# Exits with status 1 if:
+#   - Failed to fetch the TOML file
+#   - The release version is not found in the TOML file
+#   - The implementation address for the specified contract is not found
+fetch_standard_address() {
+    local network_name="$1"
+    local release_version="$2"
+    local contract_name="$3"
+
+    # Determine the correct toml url
+    local toml_url="https://raw.githubusercontent.com/ethereum-optimism/superchain-registry/a54d203e727f1cdeff06b883e239febf92b4b1d6/validation/standard/standard-versions"
+    if [ "$network_name" = "mainnet" ]; then
+        toml_url="$toml_url.toml"
+    elif [ "$network_name" = "sepolia" ]; then
+        toml_url="$toml_url-sepolia.toml"
+    else
+        echo "Error: NETWORK must be set to 'mainnet' or 'sepolia'" 1>&2
+        exit 1
+    fi
+
+    # Fetch the TOML file content from the URL if not already cached for this URL
+    if [ -z "${CACHED_TOML_CONTENT[$toml_url]:-}" ]; then
+        CACHED_TOML_CONTENT[$toml_url]=$(curl -s "$toml_url")
+        # shellcheck disable=SC2181
+        if [ $? -ne 0 ]; then
+            echo "Error: Failed to fetch TOML file from $toml_url" 1>&2
+            exit 1
+        fi
+    fi
+
+    # Use the cached content for the current URL
+    local toml_content
+    toml_content="${CACHED_TOML_CONTENT[$toml_url]}"
+
+    # Find the section for required contracts release
+    local section_content
+    section_content=$(echo "$toml_content" | awk -v version="$release_version" '
+        $0 ~ "^\\[releases.\"op-contracts/v" version "\"\\]" {
```

Comment:
We recently did some clean up in the registry to un-nest the tables from under "releases" https://github.com/ethereum-optimism/superchain-registry/pull/768. 

You'll need a fix equivalent to this https://github.com/ethereum-optimism/optimism/pull/13412/commits/0e054582b9a961be958d54d3653dff7df8f6e235